packages/driver/src/multi-domain/communicator.ts

```diff
@@ -132,7 +103,7 @@ export class SpecBridgeDomainCommunicator extends EventEmitter {
 
     try {
       // We always want to make sure errors are posted, so clean it up to send.
-      const preProcessedError = preprocessErrorForPostMessage(err)
+      const preProcessedError = _.isError(err) ? preprocessErrorForPostMessage(err) : err
```

Comment:
This becomes a problem if someone tries to throw something that is unserializable. Do we think this is a valid concern?

Reply:
I think it is a valid concern. In general i think we should add a basic try-catch around postMessage for any unserializable data, so we can at least send a message with an error.